src/script/components/MessagesList/Message/ContentMessage/asset/LinkPreviewAssetComponent.tsx

```diff
@@ -123,6 +128,7 @@ const LinkPreviewAsset: React.FC<LinkPreviewAssetProps> = ({header = false, mess
                 rel="noopener noreferrer"
                 title={preview.url}
                 data-uie-name="link-preview-url"
+                ref={linkRef}
```

Comment:
Question: I'm wondering is a simple `onclick => event.stopPropagation()` would work?